common/djangoapps/xblock_django/models.py

```diff
@@ -40,3 +42,42 @@ def disabled_block_types(cls):
             return ()
 
         return config.disabled_blocks.split()
+
+
+class XBlockDeprecatedAdvancedComponentConfig(ConfigurationModel):
```

Comment:
@ehteshamkafeel I agree that keeping them separate would make sense if we needed to enable/disable them separately. However, in our use-case we will only be adding to the two lists over time and not disabling either of them completely. If some specific XBlock no longer needs to be disabled, only it will be removed from the list.